portal/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -0,0 +1,115 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import cromwell.pipeline.datastorage.dto.{ User, UserId, UserNoCredentials }
+import cromwell.pipeline.service.UserService
+import cromwell.pipeline.utils.auth.{ SecurityDirective, TestUserUtils }
+import cromwell.pipeline.{ AuthConfig, ExpirationTimeInSeconds }
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+import org.mockito.Mockito._
+import org.scalatest.{ AsyncWordSpec, Matchers }
+import org.scalatestplus.mockito.MockitoSugar
+import pdi.jwt.JwtAlgorithm
+
+import scala.concurrent.Future
+
+class UserControllerTest extends AsyncWordSpec with Matchers with MockitoSugar with ScalatestRouteTest {
+
+  private val userService = mock[UserService]
+  private val userController = new UserController(userService)
+
+  "UserController" when {
+    "deactivateByEmail" should {
+      "return email and false value if user was successfully deactivated" in {
+        val dummyUser: User = TestUserUtils.getDummyUser()
+        val deactivateUserByEmailRequest = "JohnDoe@cromwell.com"
+        val response = UserNoCredentials.fromUser(dummyUser)
+
+        when(userService.deactivateByEmail(deactivateUserByEmailRequest)).thenReturn(Future(Option(response)))
+
+        Delete("/users/deactivate", deactivateUserByEmailRequest) ~> userController.route ~> check {
+          responseAs[UserNoCredentials] shouldBe response
+          status shouldBe StatusCodes.OK
+        }
+      }
+      "return server error if user deactivation was failed" in {
+        val deactivateUserByEmailRequest = "JohnDoe@cromwell.com"
```

Comment:
No need to define it in every test case. In could be defined in the outer scope.